CHANGELOG.md

```diff
@@ -1,5 +1,50 @@
 # Release Notes
 
+## Nerves v0.9.0
+
+* Update Notes
+
+Starting in Nerves v0.9.0 artifacts will no longer be fetched during
+`mix compile`. Artifact archives are intended to be fetched following `mix deps.get`
+To handle this, you will need to update your installed version of `nerves_bootstrap` by
+callin `mix nerves.local`. After updating `nerves_bootstrap`, you should update
+your mix file to include calls to the new required Nerves mix aliases by replacing
```

Comment:
I'd say `mix.exs file` instead of `mix file` since that's kind of colloquial/conversational.